chatbot.py

```diff
@@ -88,4 +88,5 @@ def message_to_bot(H,clf,learn_response):
             location = proper_nouns.pop()
             if subj[0] == "geocoding" or subj[0] == location:
                 geocoding(location)
+                mapsstatic(location)
```

Comment:
I understand that you added this here for testing purposes - but you missed importing `mapsstatic` in `chatbot.py`. Additionally, I would recommend not having this, we will have the correct flow that calls this method.

Reply:
@vishakha-lall I would like to know when and would the function be called? Preferably, it should be called during geocoding, since the function returns only an image indicating the queried location. However, I have resolved the changes.

Reply:
This method would be called in a newer flow of the bot (after a new combination NLP extracted POS tags, that is yet to be done), calling this with geocoding would be redundant. But we'll figure that out soon, for starters, we only needed this to be working for when we address that issue.